### DISCUSSION TITLE
Common Name Only

### DISCUSSION THREAD

**Discussion by karabelnikov** [bnhf/openvpn-admin-plus#43] [Q&A]
**@bnhf** 

Scott, I suggest moving the project to use **set_var EASYRSA_DN=cn_only** instead of **set_var EASYRSA_DN=org**

```
And don't use variables:
- COUNTRY=${COUNTRY}
- PROVINCE=${PROVINCE}
- CITY=${CITY}
- ORG=${ORG}
- EMAIL=${EMAIL}
- OU=${OU}
```

I consider it redundant. Since information other than CN is essentially static, we no longer display it in our certificate table.
And the OpenVPN installation scripts, in particular, PiVPN uses CN_only.

**Comment by bnhf** [comment]
@karabelnikov 

Shura, it's been a while, but my recollection is that those values add additional "randomness" to the creation of certificates and keys. Without them, I believe the encryption would be slightly easier to crack. If PiVPN doesn't use them anymore, or maybe more importantly EasyRSA, then we could probably eliminate them. Like so many things, I'd need to look at the code to be sure getting rid of them won't cause other issues.
